R/prog_com.R

```diff
@@ -20,7 +20,7 @@
 #'
 #' @keywords internal
 #' @family prog_com
-check_git_status <- function(code, dir = getwd()){
+check_clean_tree <- function(code, dir = getwd()){
   # input validation
   # TODO might want to check whether code works
   checkmate::assert_directory_exists(dir)
```

Comment:
You should always be in a git repo, the actions run `git clone` to get the code, so it will be a git repo.

I think it the way I would do this is to use `git status --porcelain` to check if there were any changes and if so then commit them. If that command produces no output nothing needs to be done. See https://stackoverflow.com/a/5139346/2055486 for some other options.

Reply:
doing this in https://github.com/r-lib/ghactions/issues/233 now

Reply:
also https://github.com/r-lib/ghactions/issues/236